packages/plugin-posclient-api/src/models/PutData.ts

```
@@ -152,25 +153,41 @@ const getArrangeProducts = async (config: IEbarimtConfig, doc: IDoc) => {
   for (const detail of (doc.details || []).filter(d => d.product)) {
     const { product } = detail;
 
-    const stock = genStock(detail, product, config);
+    const taxRule = product.taxRules && product.taxRules[posToken] || {};
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)